src/panels/config/hardware/ha-config-hardware.ts

```diff
@@ -287,26 +286,24 @@ class HaConfigHardware extends SubscribeMixin(LitElement) {
                   </div>
                   ${documentationURL
                     ? html`
-                        <ha-list>
-                          <ha-clickable-list-item
-                            .href=${documentationURL}
-                            open-new-tab
-                            twoline
-                            hasMeta
+                        <ha-md-list-item
```

Comment:
Missing a `<ha-md-list>` wrapper here. Is that on purpose?

Reply:
Yeah, just one item, so no need for a list